src/main/java/org/assertj/core/util/Strings.java

```diff
@@ -127,6 +127,15 @@ public static StringsToJoin join(Iterable<?> toStringable) {
     return new StringsToJoin(strings);
   }
 
+  /**
+   * Converts given vararg to string array
+   * @param values {@code CharSequence} vararg
+   * @return converted to array values
+   */
+  public static CharSequence[] toArray(CharSequence... values) {
+    return stream(values).map(CharSequence::toString).toArray(String[]::new);
+  }
```

Comment:
Not needed, `values` is already considered as an array (automatically created by the compiler)